packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -580,3 +580,50 @@ class _UiKitPlatformView extends LeafRenderObjectWidget {
     renderObject.updateGestureRecognizers(gestureRecognizers);
   }
 }
+
+/// A helper widget provides the implementer of a new platform view certain utilities.
+///
+/// The widget is responsible to handle painting, gestures and semantics.
+class PlatformViewSurface extends LeafRenderObjectWidget {
+
+  /// Construct a `PlatformViewSurface`.
+  ///
+  /// The [id] and the [controller] must not be null.
+  /// By default, this widget will create a `PlatformViewLayer` with rect = size & offset and the same id of this `PlatformViewSurface`in the layer tree. If you want to use
+  /// a different type of layer (e.g.Android), or a different configuration(e.g. different sizes), implement [customLayerFactory] to have a customized layer created instead.
```

Comment:
Ditto on length, as well as missing spaces for `(e.g. Android)`.  We should avoid parenthesis anyway in doc comments, maybe

`a different type of layer, such as Android, or a different configuration, such as supporting different sizes, provide a [customLayerFactory].`